src/screens/FeedScreen/Content.js

```diff
@@ -23,6 +23,7 @@ import useCalculationContent from './hooks/useCalculationContent';
 import {getCommentLength} from '../../utils/getstream';
 
 const {width: screenWidth} = Dimensions.get('window');
+const BUFFER_CONTENT_TEXT_HEIGHT = 50;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The constant `BUFFER_CONTENT_TEXT_HEIGHT` is introduced but there's no explanation or comment about what it represents. It would be helpful to add a comment explaining its purpose for better maintainability.
```diff
+ // BUFFER_CONTENT_TEXT_HEIGHT is used to adjust the height of the content container
  const BUFFER_CONTENT_TEXT_HEIGHT = 50;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->